src/Directory.Build.props

```diff
@@ -5,6 +5,11 @@
   <!-- This file is imported by all projects at the beginning of the project files -->
   <Import Project="$(RepoToolsetDir)Settings.props" />
 
+  <PropertyGroup>
+    <IncludeSymbols>true</IncludeSymbols>
```

Comment:
Currently nothing, but I figure uploading them to MyGet for indexing might help someone debug one of the projects down the line.